README.md

```diff
@@ -35,29 +35,42 @@ pip install neural-compressor
 > More installation methods can be found at [Installation Guide](https://github.com/intel/neural-compressor/blob/master/docs/source/installation_guide.md). Please check out our [FAQ](https://github.com/intel/neural-compressor/blob/master/docs/source/faq.md) for more details.
 
 ## Getting Started
-### Quantization with Python API
 
-```shell
-# Install Intel Neural Compressor and TensorFlow
-pip install neural-compressor
-pip install tensorflow
-# Prepare fp32 model
-wget https://storage.googleapis.com/intel-optimized-tensorflow/models/v1_6/mobilenet_v1_1.0_224_frozen.pb
+### Weight-Only Quantization
+```python
+# Get the float model
+from transformers import AutoModel
+
+float_model = AutoModel.from_pretrained("bert-base-uncased")
```

Comment:
Is this a good example??

Reply:
How about use `facebook/opt-1.3b`? For getting started demo, we prefer not to utilize a model that is too large, as it may take quite a while to load. The  `facebook/opt-1.3b`(about 2.6 GB) is one of the OPT-series.

Reply:
llama2-7b or mistral-7b

opt is not popular.

Reply:
okay, updated it to `mistralai/Mistral-7B-v0.1`.